packages/framework/ai-collab/README.md

```diff
@@ -1,34 +1,194 @@
+- Calender App: "Manage my calender to slot in a new 2:30 appointment"
+
+## Usage
+
+### Your SharedTree types file
+This file is where we define the types of our task management application's SharedTree data
+```ts
+//  --------- File name: "types.ts" ---------
+import { SchemaFactory } from "@fluidframework/tree";
+
+const sf = new SchemaFactory("ai-collab-sample-application");
+
+export class Task extends sf.object("Task", {
+	title: sf.required(sf.string, {
+		metadata: { description: `The title of the task` },
+	}),
+	id: sf.identifier,
+	description: sf.required(sf.string, {
+		metadata: { description: `The description of the task` },
+	}),
+	priority: sf.required(sf.string, {
+		metadata: { description: `The priority of the task in three levels, "low", "medium", "high"` },
+	}),
+	complexity: sf.required(sf.number, {
+		metadata: { description: `The complexity of the task as a fibonacci number` },
+	}),
+	status: sf.required(sf.string, {
+		metadata: { description: `The status of the task as either "todo", "in-progress", or "done"` },
+	}),
+	assignee: sf.required(sf.string, {
+		metadata: { description: `The name of the tasks assignee e.g. "Bob" or "Alice"` },
+	}),
+}) {}
+
+export class TaskList extends sf.array("TaskList", SharedTreeTask) {}
+
+export class Engineer extends sf.object("Engineer", {
+	name: sf.required(sf.string, {
+		metadata: { description: `The name of an engineer whom can be assigned to a task` },
+	}),
+	id: sf.identifier,
+	skills: sf.required(sf.string, {
+		metadata: { description: `A description of the engineers skills which influence what types of tasks they should be assigned to.` },
+	}),
+	maxCapacity: sf.required(sf.number, {
+		metadata: { description: `The maximum capacity of tasks this engineer can handle measured in in task complexity points.` },
+	}),
+}) {}
+
+export class EngineerList extends sf.array("EngineerList", SharedTreeEngineer) {}
+
+export class TaskGroup extends sf.object("TaskGroup", {
+	description: sf.required(sf.string, {
+		metadata: { description: `The description of the task group, which is a collection of tasks and engineers that can be assigned to said tasks.` },
+	}),
+	id: sf.identifier,
+	title: sf.required(sf.string, {
+		metadata: { description: `The title of the task group.` },
+	}),
+	tasks: sf.required(SharedTreeTaskList, {
+		metadata: { description: `The lists of tasks within this task group.` },
+	}),
+	engineers: sf.required(SharedTreeEngineerList, {
+		metadata: { description: `The lists of engineers within this task group which can be assigned to tasks.` },
+	}),
+}) {}
+
+export class TaskGroupList extends sf.array("TaskGroupList", SharedTreeTaskGroup) {}
+
+export class PlannerAppState extends sf.object("PlannerAppState", {
+	taskGroups: sf.required(SharedTreeTaskGroupList, {
+		metadata: { description: `The list of task groups that are being managed by this task management application.` },
+	}),
+}) {}
+```
+
+### Example 1: Collaborate with AI
+```ts
```

Comment:
```suggestion
### Example 1: Collaborate with AI

```ts
```

Reply:
Updated in commit b700ced170a403db24c887b25ce4917392948b3c